### PR TITLE
chore(project): draft

### PR DESCRIPTION
**Pull request by bas080**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,44 @@
...
+
+## Quick start
+
+For this example we will use aurelia-c3 which basicly wraps the c3 library.
+
+1. `jspm install aurelia-graphs aurelia-graphs-c3`
+2. .plugin('aurelia-graphs') and .plugin('aurelia-graphs-c3');
+3. create a model with definitions and data (how to do this depends on the
+   graph library one is using)
+3. `<graph-element settings.bind="vmSettings"
```
Comment (RWOverdijk) [inline README.md]: 4?

**Comment by RWOverdijk** [comment]
@bas080 What else needs to be done to publish this?

**Comment by bas080** [comment]
@RWOverdijk, It would be nice to have a more extensive example of the charts in swan example. At the same time I can check if everything works as expected. Also, the dimensions-picker and the chart-picker have not been documented.

**Comment by RWOverdijk** [comment]
Ack. How long would it take you to do so?

**Comment by bas080** [comment]
Half a day.

**Pull request merged**